proto/v1/cdk.proto

```diff
@@ -52,3 +60,22 @@ message HoneyventRequest {
   int64 duration_ms = 4;
 }
 
+message WriteCacheRequest {
+  string key = 1;
+  bytes data = 2;
+  google.protobuf.Timestamp expires = 3;
+}
+
+message ReadCacheRequest {
+  string key = 1;
+}
+
+message ReadCacheResponse {
+  bool miss = 1;
+  optional bytes data = 2; // body is empty if cache miss
```

Comment:
nit: same
```suggestion
  bool hit = 1;
  optional bytes data = 2; // body is empty if cache miss (hit == false)
```

Reply:
Will revise.

Reply:
Fixed in cf85c3009a3e40c0df3c2d526721c3f99b8a56be